python/taichi/examples/rendering/simple_texture.py

```diff
@@ -0,0 +1,50 @@
+from taichi.examples.patterns import taichi_logo
+
+import taichi as ti
+
+ti.init(arch=ti.vulkan)
+
+res = (512, 512)
+pixels = ti.Vector.field(3, dtype=float, shape=res)
+
+tex_format = ti.u8
+tex = ti.Texture(tex_format, 1, (128, 128))
+tex_ndarray = ti.ndarray(tex_format, shape=(128, 128))
+
+
+@ti.kernel
+def make_texture(arr: ti.types.ndarray()):
+    for i, j in ti.ndrange(128, 128):
+        ret = taichi_logo(ti.Vector([i, j]) / 128)
+        ret = ti.cast(ret * 255, ti.u8)
+        arr[i, j] = ret
+
+
+make_texture(tex_ndarray)
+tex.from_ndarray(tex_ndarray)
+
+
+@ti.kernel
+def paint(t: ti.f32, tex: ti.types.texture):
```

Comment:
nit: Should this be `ti.types.texture()` (i.e. a function call)? Because we might need to add arguments to `texture` in the future, e.g. `format`, `dim`, ...